csportal/csportal/urls.py

```diff
@@ -16,6 +16,9 @@
 from django.conf.urls import url
 from django.contrib import admin
 
+from portal import views
+
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^', views.home)
```

Comment:
@Alir3za97 : This regex will match everything. To only match `/` you should use the *end of string anchor* which is `$`. So it should be read `r'^$'`.